.github/workflows/mobile-deploy-auto.yml

```
@@ -0,0 +1,94 @@
+    if: github.event.pull_request.merged == true && !contains(github.event.pull_request.labels.*.name, 'no-deploy')
+    runs-on: ubuntu-latest
+    outputs:
+      should_deploy: ${{ steps.check.outputs.should_deploy }}
+      deployment_track: ${{ steps.check.outputs.deployment_track }}
+      version_bump: ${{ steps.check.outputs.version_bump }}
+      platforms: ${{ steps.check.outputs.platforms }}
+    
+    steps:
+      - name: Check deployment conditions
+        id: check
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)